RecoTracker/FinalTrackSelectors/plugins/TrackListMerger.cc

```diff
@@ -215,6 +218,7 @@ TrackListMerger::TrackListMerger(edm::ParameterSet const& conf) {
   lostHitPenalty_ = conf.getParameter<double>("LostHitPenalty");
   indivShareFrac_ = conf.getParameter<std::vector<double>>("indivShareFrac");
   std::string qualityStr = conf.getParameter<std::string>("newQuality");
+  priorityToken = esConsumes<TrackAlgoPriorityOrder, CkfComponentsRecord>(edm::ESInputTag("", priorityName_));
```

Comment:
The line `priorityName_ = conf.getParameter<std::string>("trackAlgoPriorityOrder");` at the end of the constructor needs to be moved to before seeting of `priorityToken`